schemas/test_cases/v0/misc.yaml

```diff
@@ -132,9 +132,15 @@
     - http://determined.ai/schemas/expconf/v0/profiling.json
   case:
     enabled: false
-    begin_on_batch: null
+    begin_on_batch: 0
```

Comment:
actually, the "valid when empty" test should also not include `begin_on_batch`, or probably `end_on_batch`.  But if they are both `null` that would be equivalent I guess.  One of the overall rules (limitations?) of expconf is that `null` and `not provided` are not really distinguishable in go.  But I do think that not having them is a bit clearer.

(even though it was I who wrote it using `null` originally...)

Reply:
This test should be valid if no fields are set whatsoever. How would I write that test case?

Reply:
I think to get an empty struct you would want:
```yaml
case: {}
```
Because what you have:
```yaml
case:
```
is actually equivalent to:
```yaml
case: null
```

And that is not a valid `profiling.json` schema at all.

Allowing `null` to match `profiling.json` would be wrong, because the "optionalness" of a struct should be declared by the parent struct that includes it, rather than by the struct itself.  So the experiment config could can say "the profiling field is either null or it matches profiling.json".  You don't need to add that test case though because it should already be tested by the minimal valid experiment test case in test_cases/v0/experiment.yaml